server/controllers/index.js

```diff
@@ -23,7 +23,10 @@ const express = require('express'),
 const status = require('./status'),
       ui = require('./ui')
 
-router.all(['/', '/status', '/livenessProbe', '/readinessProbe'], status)
+router.all(['/status', '/livenessProbe', '/readinessProbe'], status)
+router.all(['/', '/multicloud'], (req, res) => {
+    res.redirect('/multicloud/welcome')
+})
```

Comment:
This looks good. I'll double check with Cluster Lifecycle before we claim the `/multicloud` path just to make sure.